service/app/controllers/notesmanagement/NotesController.java

```diff
@@ -31,27 +29,16 @@
     try {
       JsonNode requestData = request().body().asJson();
```

Comment:
Can you simplify this further by adding a new method (for instance, handlePostRequest) in BaseController?

public Promise<Result> createNote() {
    handlePostRequest(ActorOperations.CREATE_NOTE.getValue(), requestData, (request)-> new NoteRequestValidator().validateCreateNoteRequest(request));
}